content/pages/enterprise.html

```diff
@@ -0,0 +1,132 @@
+{# View this page at http://localhost:8080/pricing/enterprise/ #}
+
+<html>
+  <head>
+    <title>Read the Docs Enterprise Plans</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="pricing/enterprise" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="Need more than our self-serve plans? Our Enterprise pricing gives you more options." />
```

Comment:
For consistency, "Enterprise plans" was used on this page a few times.

```suggestion
    <meta name="description" content="Need more than our self-serve plans? Our Enterprise plans gives you more options." />
```

Though I also haven't considered enterprise contracts as a singular plan, as we customize the plan every time. So don't give it a proper name when talking about these plans, just "enterprise plans" or "enterprise-level" plans. Both make some sense though.